### ISSUE TITLE
gui: save pagination settings

### ISSUE THREAD

**Issue by generalmanager** [desec-io/desec-stack#543]
I noticed that the sort by setting is consistent between sessions, however the pagination setting seems to be forgotten after starting a new session. I would very much appreciate this to be saved.
Ideally this would be two values. One for the domain view and one for the token view.
Usecase: I have several TLDs with numerous DNS records and hate to not find something just to then note that severals records are on page 2, 3 etc.

**Comment by peterthomassen** [comment]
> I noticed that the sort by setting is consistent between sessions

I would be surprised if that is the case, as we do not store any cookies. I also just checked (sorted domains by "published", logged out, logged in), and the ordering did not survive the session change.

Are you sure that is the case?

> Usecase: I have several TLDs with numerous DNS records and hate to not find something just to then notice that severals records are on page 2, 3 etc.

Have you tried the search functionality?

**Comment by generalmanager** [comment]
You're right. I tried it again and it seems that the sort order I chose was rather similar to the default, so I was under the impression that my settings were saved.
With that in mind I'd like to add the sort by setting to this feature request as it seems like a significant improvement. Especially when managing several domains with lots of subdomains each.

The search feature isn't necessarily helpful for this use case. Whenever I click on a domain I'd first like to get a general overview of every subdomain, the types of DNS records etc. I'm usually not just looking for a specific subdomain. And when I am, I don't necessarily remember the exact name (whether it's written with a dash, a subsubdomain with dots etc)
